build_tools/build_ext.py

```diff
@@ -25,8 +25,11 @@
     get_frameworks,
     cuda_path,
     get_max_jobs_for_parallel_build,
+    install_and_import,
 )
 
+install_and_import("pybind11[global]")
```

Comment:
This installation should be guarded depending on which framework the extensions are being built for